extensions/shared/external-media/sources/with-media.js

```diff
@@ -217,11 +219,26 @@ export default function withMedia() {
 					<Modal
 						onRequestClose={ onClose }
 						title={ title }
+						aria={ {
+							describedby: 'jetpack-external-media-browser__description',
+						} }
 						className={ classes }
 					>
 						<div ref={ this.modalRef }>
 							{ noticeUI }
 
+							<p
+								id="jetpack-external-media-browser__description"
+								class="jetpack-external-media-browser--visually-hidden"
+							>
+								{ isCopying
+									? __(
+											'When the media is finished copying and inserting, you will be returned to the editor.',
+											'jetpack'
```

Comment:
Looks like this got indented one too many levels?

```suggestion
										'When the media is finished copying and inserting, you will be returned to the editor.',
										'jetpack'
```